src/ray/core_worker/store_provider/plasma_store_provider.cc

```diff
@@ -144,4 +148,45 @@ Status CoreWorkerPlasmaStoreProvider::Delete(const std::vector<ObjectID> &object
   return raylet_client_->FreeObjects(object_ids, local_only, delete_creating_tasks);
 }
 
+bool CoreWorkerPlasmaStoreProvider::IsException(const plasma::ObjectBuffer &buffer) {
+  // TODO (kfstorm): metadata should be structured.
+  std::string metadata = buffer.metadata->ToString();
+  auto error_type_descriptor = ray::rpc::ErrorType_descriptor();
+  for (int i = 0; i < error_type_descriptor->value_count(); i++) {
+    auto error_type_number = error_type_descriptor->value(i)->number();
```

Comment:
`metadata`,`error_type_descriptor`, `error_type_number` both should be const

Reply:
OK.